qemu/tests/cfg/device_option_check.cfg

```diff
@@ -40,7 +40,8 @@
                     ide, ahci:
                         force_drive_format = ide-drive
                     virtio_scsi, spapr_vscsi:
-                        force_drive_format = scsi-disk
+                        force_drive_format = scsi-hd
+                        force_drive_format_cd1 = scsi-cd
```

Comment:
For compatibility with previous versions,  suggest to ues `required_qemu = [x.x.x, )`. or add a host filter, like:
```
RHEL.8:
    required_qemu = [x.x.x, )
```

Reply:
Not need to compatibility with previous versions, scsi-hd and scsi-cd already exist long time, it should be used in previous release.